android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonFilePicker.kt

```diff
@@ -73,6 +73,6 @@ class AddonFilePicker(
 internal open class AddonOpenDocument : ActivityResultContracts.OpenDocument() {
     override fun createIntent(context: Context, input: Array<String>): Intent {
         // We're restricting the mime type to XPI files, because that's expected by GeckoView currently.
```

Comment:
I suppose we can remove this comment now.

Reply:
😅

Reply:
Updated!